packages/amplify-category-notifications/src/channel-FCM.ts

```diff
@@ -1,14 +1,20 @@
-const inquirer = require('inquirer');
```

Comment:
just a note.. a lot of duplicate logic in channel-APNS/Email/FCM files

Reply:
agree, but to limit the impact of migrating from JS to TS I tried to leave the logic close to what it was